Doc/library/typing.rst

```diff
@@ -599,6 +599,7 @@ The module defines the following classes, functions and decorators:
 
    A generic version of :class:`collections.deque`.
 
+   .. versionadded:: 3.5.4
    .. versionadded:: 3.6.1
```

Comment:
Why is there 2 "versionadded" here ? was this added both ein 3.5.4 and 3.6.1 ? Meaning it's not in 3.6.0 ?
If that's the case that might look like a typo in the docs from a casual reader, and might need a  clarification paragraph.

Reply:
that's correct! it _**is**_ pretty weird!

Reply:
Yeah, the story with versions is related to the fact that `typing` module used to be provisional, so features were added at random time, even between minor releases.